Awesome dev - Release of lesson version 1.0 (#1)
carpentries-incubator/scientific-metadata: instructors/general-lesson-information.Rmd

---
title: General Lesson Information
---

<div class="overview card">
  <h2 class="card-header">Overview</h2>
  <div class="row g-0">
  |  | |
  | ------------ | -------------------------------------------------------- |

>   |  | :construction: **UNDER CONSTRUCTION** :construction: <br>will be adapted to `schema.org/Course` |

Reviewer: TODO

>   | **Title** | Fundamentals of scientific metadata: why cotext matters |

Reviewer: cotext -> context

Reviewer: fixed

  | **Abstract** | URL |
  |**Event format**| webinar / workshop |

>   |**Environment**| online, breakout rooms required|

Reviewer: why not "online or presence"? the requirement about "group work rooms for tasks" still can be added

Reviewer: changed in latest commit

  | **Title** | Fundamentals of Scientific Metadata: Why Context Matters |
  | **Abstract** | URL |
  |**Event format**| webinar / workshop |
  |**Environment**| online, makes use of breakout rooms (not adjusted/tested in physical setting yet) |
  |**Scientific discipline**| generic (HMC Hub Information) |
  |**Level**| beginner |
  |**Target group**| PhD candidates and early career postdocs, little prior knowledge of metadata (annotation), RDM - Bloom Level 1 / 2, participants do not know each other |
  |**Duration** | 420 min learning time, 4 breaks of 15 min |
  |**Group size**| 15 (recommended for 2 instructors) |
  |**Prior knowledge**| none |
  |**Required equipment** | text editor, web browser |
  |**License** | https://creativecommons.org/licenses/by/4.0/ |
  |**Creator**| Silke Christine Gerlich, https://orcid.org/0000-0003-3043-5657 |
  |**Creator**|  Annika Strupp, https://orcid.org/0000-0002-0070-4337 |
  </div>
</div>

<br><br>

<img src="fig/dearInstructors.png/" height="75" align="left">
<br><br><br><br>

thank you for your interest in our training course **"Fundamentals of Scientific Metadata"**. 

> On this page, we would like to give you a brief overview on the course material. Detailed teaching instructions can be found in the <a href="instructor-guide.html" target="_blank">instructor guide</a>.

Reviewer 2: ```suggestion
On this page, we would like to give you a brief overview of the course material. Detailed teaching instructions can be found in the <a href="instructor-guide.html" target="_blank">instructor guide</a>.
```

This lesson was designed to be taught on two consecutive days, spanning 4 hours a day.
A detailed schedule is included in the instructor guide.

<br>

### Instructor Guide
In the instructor guide, we will provide you with all the detailed information on lesson structure and schedule, additional content, and recommendations necessary to teach this lesson. 

> Also included in the instructor guide, you can find `talking points` for lectures, challenges, live coding and other content. Lecture talking points contain download links for the accompanying slide sets. Of course you are free to use your own slides and take our talking points as a guide line. All of these `Talking points` can also be found in the instructor notes of the individual episodes.

Reviewer 2: ```suggestion
Also included in the instructor guide, you can find `talking points` for lectures, challenges, live coding and other content. Lecture talking points contain download links for the accompanying slide sets. Of course, you are free to use your own slides and take our talking points as a guideline. All of these `Talking points` can also be found in the instructor notes of the individual episodes.
```

**Examples of talking points:**
(<i>Additional information for instructors is written in italics</i>)

::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::: instructor
## Lecture: This is a talking points example for a lecture

**Slide set**: LINK (<i>Download link to lecture slide set</i>)

**Time estimate**: XX min. (<i>approx. time needed to deliver this content<i>)

> **Talking points**:

Reviewer 2: The example of talking points is not clear to me. 

I didn't get the example about slide 2 and slide 5, when you say `in this case, ending with slide 5.` Where do I know in this case it is about slide 5? Well, it will probably be more clear when I read the actual talking points 😁

**Group handout Markdown:** <a href="../files/exampleGroupHandout.txt" target="_blank">exampleGroupHandout.md</a> (optimised for HedgeDoc) <br>

<a href="../files/exampleGroupHandoutPreview.png" target="popup" onclick="window.open('../docs/files/exampleAhaslidesPid.png,'popup','width=1200'); return false;" onMouseOver="document.groupHandout.src='../../docs/files/CLICKexampleGroupHandoutPreview.png';" onMouseOut="document.groupHandout.src='../../docs/files/exampleGroupHandoutPreview.png';"><span><img src="files/exampleGroupHandoutPreview.png" alt="Screenshot of the HedgeDoc Markdown group handout." name="groupHandout"/></span></a>

<br>

### Example Data
The lesson starts by introducing a barely annotated dataset, which operates as the base of the whole lesson. Throughout the episodes, the learners will gradually be supplied with more information about this dataset, annotate it with the given information and continuously improve the accompanying metadata to improve its findability, accessibility, interoperability and reusability.

> This lesson was designed for a heterogenous group of learners from diverse research areas. Hence we decided to use a dataset, that is unlikely to appear in the scientific context, yet has the possibility to be applied as such.<br>

Reviewer 2: ```suggestion
This lesson was designed for a heterogeneous group of learners from diverse research areas. Hence, we decided to use a dataset that is unlikely to appear in the scientific context, yet has the possibility to be applied as such.

```

<a href="../files/exampleDataObjectColor.png" target="popup" onclick="window.open('../docs/files/exampleDataObjectColor.png,'popup','width=1200'); return false;" onMouseOver="document.exampleData.src='../../docs/files/CLICKexampleDataObjectColor.png';" onMouseOut="document.exampleData.src='../../docs/files/exampleDataObjectColor.png';"><span><img src="files/exampleDataObjectColor.png" alt="Screenshot of example data object." name="exampleData"/></span></a>

<br>

### Learning Objectives
Our learning objectives follow the model of <a href="https://archive.org/details/taxonomyofeducat0000bloo_o9o7" target="_blank">Bloom <i>et al.</i> (1956)</a> and use the measurable verbs proposed by <a href="https://doi.org/10.3390/educsci6040037" target="_blank">Stanny <i>et al.</i> (2016)</a>.

<br>

### Episode 0: Housekeeping
In the Instructor guide, we included sections labelled "Episode 0: Housekeeping". These sections are not directly related to the lesson subject, but are important (to us) for the overall teaching experience. Episode 0 sections include activities like learner warm-ups and feedback and can be found in the detailed lesson schedule.

<br>

### Lightning Talk

> We made very good experiences with closing the lesson with a **lightning talk**, that focusses on the application of structured metadata in research, gives an insight into data annotation with metadata in practice, or demonstrates a recent (scientific) software development in the metadata field. For this section, we usually invite an external speaker and assign **45 min** including discussions to the schedule of day 2. 

Reviewer 2: ```suggestion
We had very good experiences closing the lesson with a **lightning talk** that either focuses on the application of structured metadata in research, gives an insight into data annotation with metadata in practice, or demonstrates a recent (scientific) software development in the metadata field. For this section, we usually invite an external speaker and assign **45 min** including discussions to the schedule of day 2. 
```

> In our 2 day approach, we end at lunch time on the second day, which allows for extra time for discussions with the speaker. The lightning talk so far got great feedback from the learners.

Reviewer 2: ```suggestion
In our 2 day approach, we end at lunchtime on the second day, which allows for extra time for discussions with the speaker. The lightning talk so far got great feedback from the learners.
```

<br>

### Get in Touch 

You want to contribute to this lesson? Please open an issue in this lesson's Github repository:<br>
<a href="https://github.com/carpentries-incubator/scientific-metadata" target="_blank">
  <span>
    <img src="fig/github.svg" xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="#383838" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" aria-hidden="true" style="float:left; padding-right:5px;"/>Github Repository
  </span>
</a>
<br>
You want to share your experiences or have questions on this lesson? You can get in touch with the creators and maintainers by mail:<br>

> <a href="mailto:HMC@fz-juelich.de?subject=[Carpentries Incubator] Fundamentals of scientific metadata - get in touch"><span><img src="fig/mail.svg" xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="#383838" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" aria-hidden="true" style="float:left; padding-right:5px;"/> HMC at fz-juelich.de</span></a>

Reviewer: why writing out the `at`?  The real email is in the HTML anyway, this does not protect from spambots. 

Either obfuscate properly, or just write `@`
